backend/routes/announcements.js

```diff
@@ -3,6 +3,13 @@ const router = express.Router();
 const { Announcements} = require('../database');
 
 // Routes
+
+router.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
```

Comment:
Please refactor line 7 to the following to stick with convention of the other route files
`router.use( ( request, response, next ) => {`

Rename `res` on lines 8 and 9 to `response`

Reply:
Done.